internal/lib/lib.go

```diff
@@ -957,7 +957,7 @@ func IsAviLBDefaultIngressClass() (string, bool) {
 }
 
 func IsAviLBDefaultIngressClassWithClient(kc kubernetes.Interface) (string, bool) {
-	ingClassObjs, _ := kc.NetworkingV1().IngressClasses().List(context.TODO(), metav1.ListOptions{})
```

Comment:
what would you do in k8s 1.22 when networkbeta1 is deprecated?

Reply:
I guess that is out of scope of this PR but we'll probably have to go with one of the following options:
- support k8s minVersion: 1.19 maxVersion: 1.22. The kubernetes community actively maintains only the last 3 release versions so we should be in line with that with this approach.
- if we plan to support k8s version 1.18 (or less) through 1.22, there will be a need to duplicate the code for both networkingv1beta1 and networkingv1. I think we could also look at casting ingresses to v1 like we did for extensions and networkingv1beta1 ingresses like so https://github.com/avinetworks/ako/blob/9235bc726579164d9dcc7737fe3b73b246d5e477/pkg/utils/ingress.go. This would have some scope for duplication as well, since clientsets to be used will still be different.
- defer maxVersion 1.22 support till all existing users are onboard with 1.19

Reply:
Just to be aware, we are already at 1.21 in AKO 1.4.1 and we will go to 1.22 with AKO 1.5.1 Looks like we have to revisit this soon? cc @monotosh-avi

Reply:
Another reason to switch to envtest soon. We won't face this issue with envtest. 
If we are switching APIs lets add a comment/ another method to track this.